code/datums/changelog_old.dm

```diff
@@ -2699,7 +2699,7 @@
 (*)Russian roulette is now more like its exciting real life counterpart.
 (-)
 (u)ProcitizenSA (kinda)
-(*)<b>ATTENTION STATION SECURITY</b>: The following is a message from SecuTech security. <i>Thank you for trying SecuTech Virtual-Reality (VSPACE) Prisoner Containment Unit�. We hope you enjoyed your trial, but your evaluation copy has expired. Please register and purchase SecuTech Virtual-Reality (VSPACE) Prisoner Containment Unit� to continue. Thank you and have a secure day. (NT vbrig will keep on keepin on)</i>
+(*)<b>ATTENTION STATION SECURITY</b>: The following is a message from SecuTech security. <i>Thank you for trying SecuTech Virtual-Reality (VSPACE) Prisoner Containment Unit�. We hope you enjoyed your trial, but your evaluation copy has expired. Please register and purchase SecuTech Virtual-Reality (VSPACE) Prisoner Containment Unit� to continue. Thank you and have a secure day. (NT vbrig will keep on keepin on)</i>
```

Comment:
here too

Reply:
not sure what's up with those entries, might just be an encoding error